plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -863,25 +872,47 @@ public static Transaction newCreateTableTransaction(TrinoCatalog catalog, Connec
         PartitionSpec partitionSpec = parsePartitionFields(schema, getPartitioning(tableMetadata.getProperties()));
         SortOrder sortOrder = parseSortFields(schema, getSortOrder(tableMetadata.getProperties()));
 
+        Transaction txn;
```

Comment:
Minor: avoid using abbreviated variable names, use `transaction` instead

Reply:
I would say txn is a very common abbr for transaction, change it to transaction though.